doc/src/installation.md

```diff
@@ -32,8 +32,8 @@ in {
     enable = true;
 
     # path to the admin password
-    # $ echo -n password > /etc/secrets/password.txt
-    passwordFile = "/etc/secrets/password.txt";
+    # $ echo -n password | argon2 SaltSalt -id -e > /etc/secrets/password.txt
```

Comment:
this will probably be the command people copy/paste, thus having a random salt would be nicer, wdyt?
```suggestion
    # $ SALT=$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 16 | head -n 1)
    # $ echo -n password | argon2 "$SALT" -id -e > /etc/secrets/password.txt
```

Reply:
how about `SALT=$(cat /dev/urandom | head -c 16 | base64)`?

Reply:
that's fine to me as well! :)